tests/ImageSharp.Tests/TestFile.cs

```diff
@@ -140,7 +138,15 @@ public Image<Rgba32> CreateImage()
         /// </returns>
         public Image<Rgba32> CreateImage(IImageDecoder decoder)
         {
-            return Image.Load(this.image.Configuration, this.Bytes, decoder);
+            return Image.Load(this.GetImage().Configuration, this.Bytes, decoder);
+        }
+
+        private Image<Rgba32> GetImage()
```

Comment:
This means that the test files are now lazy loaded. No more mental CPU after every build!